src/plot_api/plot_api.js

```diff
@@ -251,7 +251,7 @@ Plotly.plot = function(gd, data, layout, config) {
         // calc and autorange for errorbars
         ErrorBars.calc(gd);
 
-        // TODO: autosize extra for text markers
+        // TODO: autosize extra for text markers and images
```

Comment:
ah thanks - comment-linked in https://github.com/plotly/plotly.js/pull/1403/commits/46962af8bdb37ee77bef8880c41baf7186f095ac